lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -79,14 +79,18 @@ $-separator-color: var(--nx-swatch-indigo-40);
 .nx-global-sidebar__product-info {
   flex: 1 1 auto;
   display: flex;
+  overflow: hidden;
+  padding-right: var(--nx-spacing-3x);
```

Comment:
I think it would be cleaner to use `margin` or `gap` for this spacing.  We try to avoid using padding except for things that are explicitly within the border box of the element in question.